### PR TITLE
Added views for timeout and unauthorised

### PR DESCRIPTION
**Pull request by davidmthornton**

### CODE REVIEW

File: test/utils/TestUtil.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  implicit lazy val mockConfig: MockAppConfig = new MockAppConfig(app.configuration)
+  implicit lazy val serviceErrorHandler: ErrorHandler = injector.instanceOf[ErrorHandler]
+
+  implicit lazy val request: FakeRequest[AnyContentAsEmpty.type] = FakeRequest()
+  implicit lazy val hc: HeaderCarrier = HeaderCarrier()
+  implicit lazy val ec: ExecutionContext = injector.instanceOf[ExecutionContext]
+
+  implicit class CSRFTokenAdder[T](req: FakeRequest[T]) {
+    def addToken: FakeRequest[T] = {
```
Comment (lewishayqa) [inline test/utils/TestUtil.scala]: Add parentheses to the end of this function name just to silence the IntelliJ warning,

File: test/views/errors/SessionTimeoutViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    s"have the correct document title" in {
+      document.title shouldBe "Your session has timed out"
+    }
+
+    s"have the correct page heading" in {
+      elementText("#content h1") shouldBe "Your session has timed out"
+    }
+
+    s"have the correct instructions on the page" in {
+      elementText("#content p") shouldBe "To manage your VAT account, you'll have to sign in using your Government Gateway ID."
```
Comment (lewishayqa) [inline test/views/errors/SessionTimeoutViewSpec.scala]: Could you add a test to check the sign in link location please

File: app/views/errors/unauthorised.scala.html [hunk 3]
```diff
@@ -20,8 +20,7 @@
 
   <h1>@messages("unauthorised.title")</h1>
   <p>
-    @messages("unauthorised.useThisService")
-    <a href="@appConfig.agentServicesGovUkGuidance">@messages("unauthorised.setUpAccount")</a>.
+    <a href="@appConfig.agentServicesGovUkGuidance">@messages("unauthorised.instructions")</a>
```
[the remark this answers is not in the document]
Comment (davidmthornton) [reply app/views/errors/unauthorised.scala.html]: I'll ask Jordan if there is a design for this page. Although it's a similar scenario to on other services he may want something specific on there.

File: test/views/errors/agent/NotAuthorisedForClientViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+      "has a hidden field for the VRN" in {
+        val input = element(Selectors.hiddenIdentifier)
+        input.attr("value") shouldBe vrn
+        input.attr("type") shouldBe "hidden"
+      }
+
+      "has a link which submits the form" in {
+        val input = element(Selectors.instructionsLink)
+        input.attr("onClick").contains("document.getElementById('agentInviteForm').submit();") shouldBe true
```
Comment (TomDudley92) [inline test/views/errors/agent/NotAuthorisedForClientViewSpec.scala]: can you remove the ; please

**Pull request merged**